content/applications/hr/referrals.rst

```diff
@@ -471,8 +308,10 @@ modified. Hover over the image being replaced to reveal a :guilabel:`✏️ (pen
 :guilabel:`🗑️ (garbage can)` icon. Click the :guilabel:`✏️ (pencil)` icon and a file explorer
 window appears. Navigate to the desired image file then click :guilabel:`Open` to select it.
 
-Once all changes have been made, click the :guilabel:`Save` button to save the changes, or click the
-:guilabel:`Discard` button to revert to the original information.
+The referral friend form automatically saves but can be saved manually at any time by clicking the
+*save manually* option, represented by a :guilabel:`(cloud upload)` icon, located in the top-left
+corner. To cancel any changes made, click the :guilabel:`✖️ (Discard all changes)` icon to delete
```

Comment:
is it an 'X' or a 'counter-clockwise arrow'?

Reply:
Still an X- I checked in the latest (Runbot 17.2)